index.bs

```diff
@@ -363,7 +363,7 @@ enum XRSessionMode {
   - A session mode of <dfn enum-value for="XRSessionMode">immersive-vr</dfn> indicates that the session's output will be given [=exclusive access=] to the [=/XR device=] display and that content <b>is not</b> intended to be integrated with the user's environment. The {{environmentBlendMode}} for {{immersive-vr}} sessions is expected to be {{opaque}} when possible, but MAY be {{additive}} if the hardware requires it.
   - A session mode of <dfn enum-value for="XRSessionMode">immersive-ar</dfn> indicates that the session's output will be given [=exclusive access=] to the [=/XR device=] display and that content <b>is</b> intended to be integrated with the user's environment. The {{environmentBlendMode}} MUST NOT be {{opaque}} for {{immersive-ar}} sessions.
 
-An <dfn>immersive session</dfn> refers to either an {{immersive-vr}} or an {{immersive-ar}} session. [=Immersive sessions=] MUST provide some level of [=viewer=] tracking, and content MUST be shown at the proper scale relative to the user and/or the surrounding environment. Additionally, [=Immersive sessions=] MUST be given <dfn>exclusive access</dfn> to the [=/XR device=], meaning that while the [=immersive session=] is not [=blurred=] the HTML document is not shown on the [=/XR device=]'s display, nor is content from other applications shown on the [=/XR device=]'s display.
+An <dfn>immersive session</dfn> refers to either an {{immersive-vr}} or an {{immersive-ar}} session. [=Immersive sessions=] MUST provide some level of [=viewer=] tracking, and content MUST be shown at the proper scale relative to the user and/or the surrounding environment. Additionally, [=Immersive sessions=] MUST be given <dfn>exclusive access</dfn> to the [=/XR device=], meaning that while the [=immersive session=] is {{XRVisibilityState/visible}} the HTML document is not shown on the [=/XR device=]'s display, nor is content from other applications shown on the [=/XR device=]'s display.
```

Comment:
May be worth noting that the HTML document may still be visible on another display during an immersive session, such as a desktop monitor.

Also may be worth discussing what the HTML document being hidden by an immersive session causes for the 2D [Document.hidden](https://developer.mozilla.org/en-US/docs/Web/API/Document/hidden) and [Document.visibilityState](https://developer.mozilla.org/en-US/docs/Web/API/Document/visibilityState) attributes.

Reply:
May be meaningful on headsets (i.e. when there is no desktop monitor to still show the HTML document) to mirror what signals the Fullscreen API sends regarding 2D document visibility.

Reply:
Per https://codepen.io/anon/pen/byXVXX on Chrome, entering and leaving Fullscreen doesn't trigger any visibility change event which is to be expected tbh. You're watching one of the element (e.g. video) in fullscreen so the page is technically visible.

Reply:
> May be worth noting that the HTML document may still be visible on another display during an immersive session, such as a desktop monitor.
> 
> Also may be worth discussing what the HTML document being hidden by an immersive session causes for the 2D [Document.hidden](https://developer.mozilla.org/en-US/docs/Web/API/Document/hidden) and [Document.visibilityState](https://developer.mozilla.org/en-US/docs/Web/API/Document/visibilityState) attributes.

Interesting question. I think if the document is still visible on another monitor you don't want to touch the visibilityState attribute on the 2D document because as you said it's "visible" and a secondary user (not the one in the HMD) would expect the page to continue rendering and working (it could be a control panel for e.g. beside the mirroring) therefore we shouldn't send such signal to developers (because they mentally map it to, my page is hidden, I need to stop doing work)

Reply:
Added a note specifying that this attribute has no bearing on the document's `visibilityState` and specified the tethered desktop headset scenario as an example. In general I would say that even on devices without an external display I'd expect showing immersive content to indicate that the document visibility state is still 'visible' because the page _is_ still showing you content, just not HTML content. (And that may change in the future when we talk about DOM layers.) This apparently would keep us inline with fullscreen, as Alexis pointed out.